File: src/py/mysql_serial.py
from gaussian import gaussianCDF
from mysql.connector import connect

try:
	conn = connect(
		host="localhost",
		user="root",
		password="root123",
		database="gaussian",
	)
	conn.autocommit = True
	cur = conn.cursor()
	cur.execute('''DROP TABLE IF EXISTS tb_py_serial; CREATE TABLE tb_py_serial(id INTEGER AUTO_INCREMENT
	PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL);''')
	for i in range(-500, 501):
		x = i / 100
		prob = gaussianCDF(0, 1, x)
		conn.next_result()
		cur.execute(
			f'INSERT INTO tb_py_serial(z_score, cumulative_distribution) VALUES ({x:.2f}, {prob:.6f})')
	conn.close()
except Exception as e:
	print(e)

File: src/js/mysql_serial.js
const { gaussianCDF } = require("./gaussian"),
  mysql = require("mysql2");

try {
  let conn = mysql.createConnection({
    host: "localhost",
    user: "root",
    password: "root123",
    database: "gaussian",
    flags: "MULTI_STATEMENTS",
  });
  conn.connect();
  conn.query(
    "DROP TABLE IF EXISTS js_serial; CREATE TABLE js_serial(id INTEGER AUTO_INCREMENT " +
      "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL);"
  );
  for (let i = -500; i <= 500; i++) {
    let x = i / 100,
      prob = gaussianCDF(0, 1, x);
    conn.query(
      `INSERT INTO js_serial(z_score, cumulative_distribution) VALUES (${x}, ${prob.toFixed(
        6
      )});`
    );
  }
  conn.end();
} catch (err) {
  console.log(err.message);
}
